VoteBot/Controllers/message_controller.py

```diff
@@ -0,0 +1,9 @@
+from VoteBot.Services.vote_service import VoteService
+
+class MessageController:
```

Comment:
If you decide/agree to make VoteService MemeVoteService then MessageController should also be renamed to MemeVoteMessageController. 

Ps. Why MessageController and not for example named VoteController instead. It does not really do anything with a message in its code. Or is this a Python thinghy that I maybe not now of yet, to call something like this a messageController?

Reply:
You are right. I called it MessageController because it is a layer between the message and the service. But it has to be service specific of course.

Reply:
I changed it to VoteMemeController